Nosenti/certificate-manager---Innocent
# Task3-KAN-34 Creating a Certificate

#### PR description
- This PR is for creating the certificate and save its data in indexDB

#### Ticket / Jira Link
- [KAN-34](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?assignee=712020%3A67b4d316-bd6f-4b16-a625-69afa0078a10&selectedIssue=KAN-34)

#### Checklist:
- [x] I have added screenshots

#### How To Test Changes
- After running the app, go to Example 1, at the start, you should not have certificates. 
- Now click on Create Certificate button and fill in the information, it shall save the information and redirects you to the table. 
- Upon refreshing the page, you should still see the all the information in the table. 

#### Screenshots:
![image](https://github.com/user-attachments/assets/11de563c-c86d-490f-9874-397fb1998142)
![image](https://github.com/user-attachments/assets/d0d53cff-91a4-445a-a599-5a6da586b80f)
